config.yaml

```diff
@@ -22,6 +22,7 @@ repos:
     url: https://github.com/coreos/afterburn
     vars:
       git_repo: afterburn
+      gitlab_repo: rust-afterburn
```

Comment:
Let's describe the contents of the repo, not the name of the hosting service.  There are two GitLab repos: the CentOS Stream one and the RHEL one.  We already have `{fedora,rhel8,rhel9}_package` variables; maybe call this one `rhel_package_repo`?

Reply:
That sounds good to me!